samples/language/LambdaExpression.qs

```diff
@@ -9,7 +9,7 @@
 namespace MyQuantumApp {
     open Microsoft.Quantum.Arrays;
     @EntryPoint()
-    operation Main() : Unit {
+    function Main() : Unit {
```

Comment:
Whoops, looks like this one should remain operation.

Reply:
Actually, I only noticed this because the CI failed (see https://github.com/microsoft/qsharp/actions/runs/9848645768/job/27191881736). Running `python build.py` with default settings includes a samples check that compiles all the algorithm and language samples, and integration tests verify the notebooks. So thankfully there is some coverage to help with these!

Reply:
Oh, I misunderstood your original feedback, sorry about that! You are right, samples do not get checked in CI with qsc for lints, just compilation. That's definitely a gap we should close. I've been working on turning samples tests into Rust unit tests (#1533), and with the incoming package support we have to chance to convert the rest. I think as part of that will be our best bet for getting that coverage.